### PR TITLE
document to sort the forms list

### PR DESCRIPTION
**Pull request by khushishikhu**
closes odk-x/tool-suite-X#250

addresses odk-x/tool-suite-X#250

#### What is included in this PR?
This is the documentation written for ordering the forms list according to the sort options given. The documentation is written using sphinx tools and the relevant screen short are also added at required place

### CODE REVIEW

File: odkx-src/spelling_wordlist.txt [hunk 1]
```diff
@@ -123,6 +123,7 @@ hotspots
 html
 http
 https
+homescreen
```
Comment (elmps2018) [inline odkx-src/spelling_wordlist.txt:126]: The term should be home screen (that is what is used in the rest of the documentation) homescreen is not correct. Please remove this addition/reverse this change.

File: odkx-src/survey-using.rst [hunk 2]
```diff
@@ -97,6 +97,26 @@ Completing the *Household Member Survey* subform returns us to the same screen w
     :alt: Survey Household Subform Two Entry
     :class: device-screen-vertical
 
+.. _survey-ordering-form:
+
+Ordering Forms list
```
Comment (elmps2018) [inline odkx-src/survey-using.rst:102]: Make sure to use title case or sentence case, Ordering Forms list is a mix (see https://docs.odk-x.org/docs-style-guide/#title-case-and-sentence-case for which to use where)

File: odkx-src/survey-using.rst [hunk 3]
```diff
@@ -97,6 +97,26 @@ Completing the *Household Member Survey* subform returns us to the same screen w
     :alt: Survey Household Subform Two Entry
     :class: device-screen-vertical
 
+.. _survey-ordering-form:
+
+Ordering Forms list 
+---------------------
+
+ All the forms listed on the homescreen can be ordered in two ways either by **TableID** or **Name**. Press the Action button (:guilabel:`⋮`) and select the :menuselection:`Sort by`.
```
Comment (elmps2018) [inline odkx-src/survey-using.rst:105]: home screen not homescreen
Action should be lower case

File: odkx-src/survey-using.rst [hunk 4]
```diff
@@ -97,6 +97,26 @@ Completing the *Household Member Survey* subform returns us to the same screen w
...
+
+ The screen will show two options as :menuselection:`TableID` and :menuselection:`Name` as described below:
```
Comment (elmps2018) [inline odkx-src/survey-using.rst:107]: The screen shot shows Table ID not TableID and I think the underlying data is actually table_id so should probably refer to Table ID in reference to the picture and explain it is the table_id (applies throughout)

File: .gitattributes [hunk 5]
```diff
@@ -1,6 +1,5 @@
 _static/vid/**/* filter=lfs diff=lfs merge=lfs -text
```
Comment (elmps2018) [inline .gitattributes:1]: Not sure what/why the changes are here or if this might cause any problems.

Comment (khushishikhu) [reply .gitattributes:1]: This file get changes when I tracked the images from git lfs. I have learned this thing from my previous issues. If there is something wrong Please make me correct

**Comment by elmps2018** [comment]
@khushishikhu looks like one of the checks failed, are you able to figure out why?

**Comment by elmps2018** [comment]
@khushishikhu aside from the failed build the revisions look good!

**Comment by khushishikhu** [comment]
No I don't know why is this build is failing. Can you please help me out how can I resolve this. Thank you!!

**Comment by elmps2018** [comment]
@wbrunette or @linl33 any insights?

**Comment by wbrunette** [comment]
@khushishikhu and @khushishikhu it was an internet connectivity issue (sometimes the Internet has issues). All I did was hit the re-run button and it started to build. The log said it couldn't get the docker image (which is available on the Internet).

**Comment by khushishikhu** [comment]
@wbrunette Thank you for letting me know the reason. Builds are passing now. Please let me know if it requires anything more or any error.

**Comment by elmps2018** [comment]
@khushishikhu this looks great! Merging now!
